about_edit.html

```diff
@@ -1 +1,154 @@
+							<div class="row">
+								<span><p><b class='data-label'>Birthday:</b></p>
+								<p class='data-value'>July 31st, 1980</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>College:</b></p>
+								<p class='data-value outlined-grey'>Hogwarts College</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>Hometown:</b></p>
+								<p class='data-value outlined-grey'>Godrick's Hollow, England</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>Currently Lives:</b></p>
+								<p class='data-value outlined-grey'>Godrick's Hollow, England</p></span>
+							</div>
+						</div>
+						<br>
+						<h2 class='basic-info-h2'>Contact Information</h2>
+						<br>
+						<div class="container-fluid">
+							<div class="row">
+								<span><p><b class='data-label'>Email:</b></p>
+								<p class='data-value'>harry_potter@hogwarts.edu</p></span>
+							</div>
+							<div class="row">
+								<span><p><b class='data-label'>Telephone:</b></p>
+								<p class='data-value outlined-grey'>555-123-4567</p></span>
+							</div>
+						</div>
+					</div>
+					<div class="col-md-7">
+						<h2 class='basic-info-h2'>Words to Live By:</h2>
+						<br>
+						<div class="container-fluid">
+							<div class="row">
+								<p class='data-label outlined-grey'>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+								tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+								quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+								consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+								cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+								proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+							</div>
+						</div>
+						<h2 class='basic-info-h2'>About Me</h2>
+						<br>
+						<div class="container-fluid">
+							<div class="row">
+								<p class='data-label outlined-grey'>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+								tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+								quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+								consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+								cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+								proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+							</div>
+						</div>						
+					</div>
+				</div>
+				<br>
+				<br>
+				<br>
+				<br>
+				<div class="row">
+					<div class="col-md-12 save-changes-field">
+						<button class="btn btn-lg btn-primary save-changes-button">Save Changes</button>
+					</div>
+				</div>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+				<br>
+			</div>
```

Comment:
You are using a lot of line breaks when you should be using CSS for vertical spacing.